index.html

```diff
@@ -170,5 +170,22 @@ <h3>skills</h3>
         </li>
       </ul>
     </section>
+
+    <section class="forms-section">
+      <form action="/my-file" method="post">
+        <fieldset>
+            <ul class="form-list">
+                <li><legend>
+                    <h2>Contact Me</h2>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                </legend></li>
+                <li><input maxlength="30" type="text" value="Yeremias" required></li>
+                <li><input type="email" value="natajayanj14@gmail.com" required></li>
```

Comment:
Please add the appropriate placeholder attribute to the `input` field for name and email, just like you have used a placeholder for `textarea` element :+1: , The placeholder attribute specifies a short hint that describes the expected value of a input field / textarea.